### ISSUE TITLE
Should we standardize statuses?

### ISSUE THREAD

**Issue by vsoch** [panoptes-organization/monitor-schema#3]
I'm poking around, and I'm seeing statuses like "Running" and "No execution." I'm wondering if you think that some list of valid statuses should be implemented in the schema, or it should be up to the implementation? It seems like something general enough that we could make a specific list, and we could even have a `GET` /m1/statuses endpoint so the client can know them in advance. E.g.,:

```
GET /m1/statuses

{"statuses": [{
      "name": "running",
      "description": "The workflow is currently running" ,
    },{
      "name": "pending",
      "description": "The workflow has not started, and is not scheduled." ,
    },{
      "name": "scheduled",
      "description": "The workflow has not started, and is scheduled." ,
    },{
      "name": "error",
      "description": "The workflow exited with an error." ,
    },{
      "name": "completed",
      "description": "The workflow has finished." ,
    },
}
```

For the above, the minimal recommended would be running, pending, and completed, and others (e.g., scheduled) might be an example of something added for a more complex system. What do you think @fgypas ? What others would there be? Given an error, how does the client see the error message?

**Comment by vsoch** [comment]
I'll include something like this in a PR soon to discuss further!

**Comment by fgypas** [comment]
@vsoch As in most cases, I agree with you. Just one question. When would a workflow take the status `scheduled`? This would be in cases when the jobs are submitted to a cluster?

**Comment by vsoch** [comment]
Yes exactly! I don't think it applies for panoptes, but you can imagine if there is some kind of scheduler, then there is a difference between pending (not scheduled) and scheduled. Scheduled would be an example of an extra status, right now the default I have are:

- completed
- running
- error
- pending

I'll submit this in a PR later today to review!
